test/mongo_util.py

```diff
@@ -1,23 +1,106 @@
 import logging
 import os
-import uuid
-from pathlib import Path
-from typing import Tuple
+
+from configparser import ConfigParser
 from datetime import datetime
+from pathlib import Path
+from typing import List, Tuple, Union
+
 from mongo_controller import MongoController
 from AbstractHandle.Utils import MongoUtil
 
-MONGO_EXE_PATH="MONGO_EXE_PATH"
-MONGO_TEMP_DIR="MONGO_TEMP_DIR"
-
-
-def get_mongo_info() -> Tuple[Path, Path]:
+# Mongo config
+TEST_CONFIG_PATH = "HANDLE_SERVICE_TEST_CFG"
```

Comment:
```suggestion
TEST_CONFIG_ENV_PATH = "HANDLE_SERVICE_TEST_CFG"
```

make it clear it's and environment variable, not an actual path

Reply:
👍

Reply:
:+1: